src/participant_network.star

```diff
@@ -56,96 +59,124 @@ def launch_participant_network(
             )
-    else:
-        fail(
-            "Unsupported fork epoch configuration, need to define either capella_fork_epoch, deneb_fork_epoch or electra_fork_epoch"
-        )
 
-    el_cl_genesis_config_template = read_file(
-        static_files.EL_CL_GENESIS_GENERATION_CONFIG_TEMPLATE_FILEPATH
-    )
+        el_cl_genesis_config_template = read_file(
+            static_files.EL_CL_GENESIS_GENERATION_CONFIG_TEMPLATE_FILEPATH
+        )
 
-    el_cl_data = el_cl_genesis_data_generator.generate_el_cl_genesis_data(
-        plan,
-        ethereum_genesis_generator_image,
-        el_cl_genesis_config_template,
-        final_genesis_timestamp,
-        network_params.network_id,
-        network_params.deposit_contract_address,
-        network_params.seconds_per_slot,
-        network_params.preregistered_validator_keys_mnemonic,
-        total_number_of_validator_keys,
-        network_params.genesis_delay,
-        network_params.max_churn,
-        network_params.ejection_balance,
-        network_params.eth1_follow_distance,
-        network_params.capella_fork_epoch,
-        network_params.deneb_fork_epoch,
-        network_params.electra_fork_epoch,
-    )
+        el_cl_data = el_cl_genesis_data_generator.generate_el_cl_genesis_data(
+            plan,
+            ethereum_genesis_generator_image,
+            el_cl_genesis_config_template,
+            final_genesis_timestamp,
+            network_params.network_id,
+            network_params.deposit_contract_address,
+            network_params.seconds_per_slot,
+            network_params.preregistered_validator_keys_mnemonic,
+            total_number_of_validator_keys,
+            network_params.genesis_delay,
+            network_params.max_churn,
+            network_params.ejection_balance,
+            network_params.eth1_follow_distance,
+            network_params.capella_fork_epoch,
+            network_params.deneb_fork_epoch,
+            network_params.electra_fork_epoch,
+        )
+    else:
+        # split up the name from dencun-devnet-12 to dencun and devnet-12
+        devnet_name = network_params.network.split("-")[0]
+        devnet_number = network_params.network.split("-")[-1]
+        el_cl_genesis_uuid = plan.upload_files(
+            src="github.com/ethpandaops/{0}-devnets/network-configs/devnet-{1}".format(
+                devnet_name, devnet_number
+            ),
+            name="el_cl_genesis",
+        )
+        el_cl_genesis_data_uuid = plan.run_sh(
+            run="mkdir -p /network-configs/ && mv /opt/* /network-configs/",
+            store=[StoreSpec(src="/network-configs/", name="el_cl_genesis_data")],
+            files={"/opt": el_cl_genesis_uuid},
+        )
+        el_cl_data = el_cl_genesis_data.new_el_cl_genesis_data(
+            el_cl_genesis_data_uuid.files_artifacts[0],
+            "0",
```

Comment:
what is this magical "0"?

Reply:
its a dummy, genesis validators root. I can actually get this value and use it.

Reply:
can we use a constant? :P "DUMMY_GENESIS_VALIDATORS_ROOT = '0'"

Reply:
will solve in #437